csi/GET_STARTED.md

```diff
@@ -82,7 +82,7 @@ Then, in the same terminal where you exported `MR_HOSTNAME`, perform the followi
 
 ```bash
 curl --silent -X 'POST' \
-  "$MR_HOSTNAME/api/model_registry/v1alpha2/registered_models" \
+  "$MR_HOSTNAME/api/model_registry/v1alpha3/registered_models" \
```

Comment:
I know that we want to be back-compatible, but we do not have yet any stable released versions should we be changing the version numbers this aggressively? may be not right issue to ask this question either :(

Reply:
Yes, we can avoid "releasing too often", at the same time I understood we wanted to be strict on bumping the version number for ever breaking change nevertheless.

Here is the short-blanket: either we minimize releases keeping the number bumps, or we release as often as needed and we relax strictness on the backward incompatibility of the REST API number. my2c.